core/src/main/java/org/apache/accumulo/core/data/LoadPlan.java

```diff
@@ -220,7 +222,7 @@ public Builder addPlan(LoadPlan plan) {
 
       @Override
       public LoadPlan build() {
-        return new LoadPlan(fmb.build());
+        return new LoadPlan(Collections.unmodifiableList(fmb));
```

Comment:
This behaviors differs because user could still call addPlan after calling build and mutate the list.